site-src/v1alpha2/contributing/enhancement-requests.md

```diff
@@ -1,34 +1,41 @@
 # Enhancement Tracking and Backlog
 
-Inspired by [Kubernetes enhancements](https://github.com/kubernetes/enhancements), service-api's
-provides a process for introducing new functionality or considerable changes to the project. The
+Inspired by [Kubernetes enhancements][enhance], Gateway API provides a process for
+introducing new functionality or considerable changes to the project. The
 enhancement process will evolve over time as the project matures.
 
-Enhancements provides the basis of a community roadmap. Enhancements may be filed by anyone, but
-require approval from a maintainer to accept the enhancement into the project.
+[enhance]: https://github.com/kubernetes/enhancements
+
+Enhancements provides the basis of a community roadmap. Enhancements may be
+filed by anyone, but require approval from a maintainer to accept the
+enhancement into the project.
 
 ## Quick start
 
-1. Create an [Issue](https://github.com/kubernetes-sigs/gateway-api/issues/new/choose) and select
-"Enhancement Request".
-2. Follow the instructions in the enhancement request template and submit the Issue.
+1. Create an [Issue][issue] and select "Enhancement Request".
+2. Follow the instructions in the enhancement request template and submit the
+   Issue.
+3. (depending on size of change) Start a [draft Gateway Enhancement Proposal
+   (GEP)][gep]
+
+[issue]: https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[gep]: /contributing/gep.md
```

Comment:
Links should not include md suffix for mkdocs. (https://deploy-preview-684--kubernetes-sigs-gateway-api.netlify.app/contributing/enhancement-requests/ to test out existing link). On a related not, the gep page is not linked in the navigation. That could be done with an update to `mkdocs.yml` but I'm not sure it's required. IT could be confusing if it were right beside this page in navigation. The link from this page to the GEP one is probably sufficient.

```suggestion
[gep]: /contributing/gep
```